swap/src/alice.rs

```diff
@@ -185,3 +184,19 @@ impl Behaviour {
         debug!("Sent Message2");
     }
 }
+
+impl Default for Behaviour {
+    fn default() -> Self {
+        let identity = Keypair::generate_ed25519();
+
+        Self {
+            pt: PeerTracker::default(),
+            amounts: Amounts::default(),
+            message0: Message0::default(),
+            message1: Message1::default(),
+            message2: Message2::default(),
+            message3: Message3::default(),
+            identity,
+        }
+    }
+}
```

Comment:
```suggestion
impl Default for Behaviour {
    fn default() -> Self {
        let identity = Keypair::generate_ed25519();

        Self {
            identity,
            ..Default::default()
        }
    }
}
```

Will do in follow-up PR